internal/template/funcmap.go

```diff
@@ -26,18 +26,22 @@ var funcMap = map[string]interface{}{
 }
 
 // get allows a template to optionally lookup a value from a dict.
-// If a value is not found, return nil.
+// If a value is not found, it will check for a single default.
+// If there is no default, it will return nil.
 //   {{if get . "OPTIONAL_KEY"}} {{.}} {{end}}  // GOOD
 //   {{.OPTIONAL_KEY}}  // BAD, will fail if the key is not set
 //
 // Keys can reference multiple levels of maps by using "." to indicate a new level
 // (e.g. L1.L2 will lookup key L1 in the top level map then L2 within the value.)
-func get(m map[string]interface{}, key string) interface{} {
+func get(m map[string]interface{}, key string, def ...interface{}) interface{} {
```

Comment:
I'd prefer "default", "def" is a little ambiguous since it's more commonly an abbreviation of "define" (like in Python).

Reply:
default is a keyword in go, any other suggestions?

Reply:
Oh, I didn't notice, you're right. That's inconvenient. Maybe `fallback`? I'll leave it up to you.

Reply:
Since these are internal funcs I am not too concerned with the var name and they're documented anyway. Resolving for now